Jenkinsfile

```diff
@@ -0,0 +1,219 @@
+pipeline {
+    agent {
+        node {
+            label 'operator-sdk'
+        }
+    }
+
+    libraries {
+        lib('fh-pipeline-library')
+    }
+    
+    environment {
+        GOPATH = "${env.WORKSPACE}/"
+        PATH = "${env.PATH}:${env.WORKSPACE}/bin"
+        GOOS = "linux"
+        GOARCH = "amd64"
+        CGO_ENABLED = 0
+        OPERATOR_NAME = "unifiedpush-operator"
+        OPERATOR_TEST_NAME = "${env.OPERATOR_NAME}-test"
+        OPENSHIFT_PROJECT_NAME = "test-${env.OPERATOR_NAME}-${currentBuild.number}-${currentBuild.startTimeInMillis}"
+        CLONED_REPOSITORY_PATH = "src/github.com/aerogear/unifiedpush-operator"
+        OPERATOR_CONTAINER_IMAGE_NAME = "quay.io/aerogear/${env.OPERATOR_NAME}:${env.BRANCH_NAME}"
```

Comment:
If I create a PR from my fork, will `${env.BRANCH_NAME}` have my name in it? I'm just thinking that if you created a PR with the same branch name, that it could overwrite my image, maybe? Or in the case of PRs, is this variable the PR number? I can't remember...

I hope it's the PR number actually, because otherwise I could just create a PR from master of my fork ;D

Reply:
Yes, fortunately it's [`PR-$PR_NUMBER`](https://quay.io/repository/aerogear/unifiedpush-operator?tab=tags) :)